lib/segment/src/index/field_index/full_text_index/text_index.rs

```diff
@@ -286,7 +286,7 @@ impl FullTextIndex {
         phrase: &str,
         hw_counter: &HardwareCounterCell,
     ) -> Option<ParsedQuery> {
```

Comment:
interestingly the signature of the function was already setup to return an `Option` and the `rustdoc` even says

> If there are any unseen tokens, returns `None`

I guess this was an oversight :)